CHANGELOG.md

```diff
@@ -5,8 +5,9 @@ All notable changes to this project will be documented in this file.
 ## Unreleased
 
 ### Added
+- [#1508](https://github.com/pints-team/pints/pull/1508) Added a method `OptimisationController.set_max_unmoved_iterations` that allows methods to stop after 1 or more iterations with no significant movement in parameter space.
```

Comment:
"unmoved/unchanged" is a bit vague and not immediately obvious to me in terms of whether it refers to xtol or ftol? `unmoved parameters`, `unchanged objective`?

Reply:
I agree! But can't think of anything better that doesn't involve deprecating `set_max_unchanged_iterations` which is probably one of the most used methods of the controller class

Reply:
Hmmmm, given all the different names that exist already, inventing another set that are pints specific doesn't seem great anyway - and even more so if they are vague!